scalastyle-config.xml

```diff
@@ -1,6 +1,6 @@
 <scalastyle commentFilter="enabled">
```

Comment:
Slightly different format to the standard, fine for it to be left though

[scalastyle-config.xml.zip](https://github.com/hmrc/ras-frontend/files/12302425/scalastyle-config.xml.zip)